app/src/main/java/com/asfoundation/wallet/ui/iab/MergedAppcoinsFragment.kt

```diff
@@ -242,17 +242,22 @@ class MergedAppcoinsFragment : DaggerFragment(), MergedAppcoinsView {
   }
 
   private fun setBonus() {
-    //Build string for both landscape (header) and portrait (radio button) bonus layout
-    appcoins_radio?.bonus_value?.text =
-        getString(R.string.gamification_purchase_header_part_2, bonus)
-    bonus_value?.text = getString(R.string.gamification_purchase_header_part_2, bonus)
-
-    //Set visibility for both landscape (header) and portrait (radio button) bonus layout
-    if (appcoins_radio_button.isChecked) {
-      bonus_layout?.visibility = VISIBLE
-      bonus_msg?.visibility = VISIBLE
+    if (bonus.isNotBlank()) {
```

Comment:
The same comment as above, regarding the possibility of using isNotEmpty instead of isNotBlank